src/proto/stationapi.proto

```
@@ -0,0 +1,228 @@
+syntax = "proto3";
+
+package app.trainlcd.grpc;
```

Buf (1.47.2)

Files with package "app.trainlcd.grpc" must be within a directory "app/trainlcd/grpc" relative to root but were in directory "src/proto".

(PACKAGE_DIRECTORY_MATCH)